presto-main/src/main/java/com/facebook/presto/cost/SimpleFilterProjectSemiJoinStatsRule.java

```diff
@@ -50,11 +60,15 @@
     private static final Pattern<FilterNode> PATTERN = filter();
 
     private final FilterStatsCalculator filterStatsCalculator;
+    private final LogicalRowExpressions logicalRowExpressions;
```

Comment:
The name here is a little confusing ...

Reply:
Could you elaborate? I thought this is the name we have agreed on.

Reply:
In the code below, you can see both LogicalRowExpression.func1 and logicalRowExpression.func2 below :) but I don't think  it matter much.